addons/l10n_pe_pos/static/src/overrides/components/partner_editor/partner_editor.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module */
+
+import { _t } from "@web/core/l10n/translation";
+import { AlertDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { PartnerEditor } from "@point_of_sale/app/screens/partner_list/partner_editor/partner_editor";
+import { patch } from "@web/core/utils/patch";
+
+patch(PartnerEditor.prototype, {
+    setup() {
+        const res = super.setup(...arguments);
+        if (!this.pos.isPeruvianCompany()) {
+            return res;
+        }
+        this.intFields.push("city_id", "l10n_latam_identification_type_id", "l10n_pe_district");
+        this.changes.city_id = this.props.partner.city_id && this.props.partner.city_id.id;
+        this.changes.l10n_latam_identification_type_id =
+            this.props.partner.l10n_latam_identification_type_id &&
+            this.props.partner.l10n_latam_identification_type_id.id ||
+            this.pos.default_l10n_latam_identification_type_id;
+        this.changes.l10n_pe_district = this.props.partner.l10n_pe_district && this.props.partner.l10n_pe_district.id;
+        return res;
+    },
+    async confirm() {
+        const city = this.pos.models["res.city"].find((city) => city.id === parseInt(this.changes.city_id));
```

Comment:
If you're not in a pe country, you end up with a traceback. 
add: if (this.pos.isPeruvianCompany())

Reply:
good catch! Thanks, I applied your suggestion.